tools/WinObjC.Packaging/WinObjC.Packaging.nuproj

```diff
@@ -17,22 +17,42 @@
   <PropertyGroup Label="Globals">
     <OutputPath>$(SolutionDir)\OutputPackages\$(Configuration)\</OutputPath>
     <ProjectGuid>ddc2aa5f-c8ae-48cb-adb7-9278985c3db7</ProjectGuid>
+    <SkipGetPackageContentsForOtherPlatforms>true</SkipGetPackageContentsForOtherPlatforms>
   </PropertyGroup>
   <PropertyGroup Label="Configuration" />
   <ItemGroup>
-    <None Include="Package.Default.props" />
-    <None Include="Package.Default.targets" />
-    <None Include="Package.props" />
-    <None Include="Package.targets" />
-    <None Include="WinObjC.Packager.props" />
-    <None Include="WinObjC.Packager.targets" />
-    <None Include="WinObjC.Packageable.props" />
-    <None Include="WinObjC.Packageable.targets" />
```

Comment:
I know these wound up in the package before. Is there custom logic in either `ImportedByThisProj.*` that needs adjusted for what you've done here?